packages/components/time-picker/__tests__/time-picker.test.tsx

```diff
@@ -143,6 +143,28 @@ describe('TimePicker', () => {
     expect(value.value).toBeInstanceOf(Date)
   })
 
+  it('should correctly cancel value after opened twice', async () => {
+    const value = ref()
+    const wrapper = mount(() => <TimePicker v-model={value.value} />)
+
+    const input = wrapper.find('input')
+    await input.trigger('blur')
+    await input.trigger('focus')
+    document
```

Comment:
```suggestion
    wrapper.find('.el-time-panel__btn.cancel')
```
Perhaps it would be more accurate to query inside the component to avoid possible conflicts with component content rendered by other test cases.

Reply:
But i'm wondering, we do this a lot of times in other tests case. Should we now start using this method if possible ?